backend/inspirehep/records/config.py

```diff
@@ -107,11 +107,13 @@
             "application/x-bibtex": f"{INSPIRE_SERIALIZERS}:literature_bibtex_response",
             "application/vnd+inspire.latex.eu+x-latex": f"{INSPIRE_SERIALIZERS}:latex_response_eu",
             "application/vnd+inspire.latex.us+x-latex": f"{INSPIRE_SERIALIZERS}:latex_response_us",
+            "text/html": f"{INSPIRE_SERIALIZERS}:literature_cv_html_response",
```

Comment:
won't this be served by default if you open the API in the browser? maybe we should have a custom one.